metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -540,19 +563,24 @@
                            tmp.concept_id = t1.concept_id AND
                            tmp.revision_id = t1.revision_id)"
                           table)]
-            efs-delete (when (not (= "efs-off" (efs-config/efs-toggle)))
+            efs-delete (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
                          (util/time-execution
                           (efs/delete-concepts provider concept-type concept-id-revision-id-tuples)))
-            oracle-delete (when (not (= "efs-only" (efs-config/efs-toggle)))
+            oracle-delete (when (not= "dynamo-only" (dynamo-config/dynamo-toggle))
+                            (util/time-execution
+                             (j/execute! conn stmt)))
+            dynamo-delete (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
                             (util/time-execution
-                             (j/execute! conn stmt)))]
+                             ))]
         (when efs-delete
           (info "Runtime of EFS force-delete-concepts: " (first efs-delete) " ms."))
         (when oracle-delete
           (info "Runtime of Oracle force-delete-concepts: " (first oracle-delete) " ms."))
+        (when dynamo-delete
+          (info "Runtime of DynamoDB force-delete-concepts: " (first dynamo-delete) " ms."))
         (if oracle-delete
           (second oracle-delete)
-          (second efs-delete))))))
+          (second dynamo-delete))))))
```

Comment:
This logic needs to be distinct between backends. The various backends needs to be broken out into multiple functions.